files/scripts/graphdb.sh

```diff
@@ -98,4 +105,87 @@ function createRepositoryFromFile {
   fi
 }
 
+function interpolate {
+  local input=""
+  while IFS='' read -r line || [ -n "$line" ]; do
+    input="$input$line"$'\n'
+  done
+
+  vars=$(echo "${input}" | grep -o "\${[^}]*}" | sed 's/[${}]//g' | uniq)
+
+  local output="$input"
+  for var in $vars; do
+    value=${!var:-''}
+    if [ -n "$value" ]; then
+      output=${output//\$\{$var\}/$value}
+    fi
+  done
+  echo "${output}"
+}
+
+function cloudBackup {
+  BACKUP_TIMESTAMP="$(date +'%Y-%m-%d_%H-%M-%S')"
+  BACKUP_NAME="graphdb-backup-${BACKUP_TIMESTAMP}.tar"
+  export BACKUP_NAME BACKUP_TIMESTAMP
+
+  local backup_options=
+  backup_options=$(interpolate < "$1")
+
+  log "Creating cloud backup ${BACKUP_NAME}"
+
+  local response=
+  local response_status
+  response=$(mktemp)
+  response_status=$(curl -X POST \
```

Comment:
post is implied when you give a request body

Reply:
I prefer some things to be explicit, like in this case. It's easier to read and understand that this is a POST before even getting to `--data-binary`.